dev/io.openliberty.mcp.internal_fat/fat/src/io/openliberty/mcp/internal/fat/tool/ToolTest.java

```diff
@@ -53,6 +53,27 @@ public static void teardown() throws Exception {
         server.stopServer();
     }
 
+    @Test
+    public void postJsonRpc() throws Exception {
+        String jsonBody = """
+                            {
+                                "jsonrpc": "2.0",
+                                "method": "tools.call",
+                                "id": 1
+                            }
+                        """;
```

Comment:
This request isn't valid (wrong method name, no params). We should make sure we make a valid request to ensure that the error we get back is due to the bad `Accept` header and not something else.